hello-oboe/app/src/main/cpp/hello-oboe.cpp

```diff
@@ -23,11 +23,17 @@ static OboeSinePlayer *oboePlayer = nullptr;
 
 
 extern "C" {
-    JNIEXPORT void JNICALL
+    /* Create Oboe playback stream
+     * Returns:  0 - success
+     *          -1 - failed
+     */
+    JNIEXPORT jint JNICALL
```

Comment:
This doesn't feel like idiomatic return values for Java. It's fine for C++, but (imo) it should be returning a jboolean set to 'true' for success and 'false' for failure. Then the code below is:

```
if (createStream()) {
}
```

Which feels much more natural to me.

Reply:
agreed and good point.  It is the point that we discussed and finally agreed upon for audio that we will return int instead of bool in favor of future compatibility ( we might want to populate future more error code types to Java side ).